src/coreclr/jit/instr.cpp

```diff
@@ -800,6 +800,71 @@ CodeGen::OperandDesc CodeGen::genOperandDesc(GenTree* op)
 
             switch (intrinsicId)
             {
+                case NI_AVX_BroadcastScalarToVector128:
+                case NI_AVX_BroadcastScalarToVector256:
+                {
+                    // we have the assumption that AVX_BroadcastScalarToVector*
+                    // only take the memory address as the operand.
+                    assert(hwintrinsic->isContained());
+                    assert(hwintrinsic->OperIsMemoryLoad());
+                    assert(hwintrinsic->GetOperandCount() == 1);
+                    GenTree* BroadcastScalar = hwintrinsic->Op(1);
```

Comment:
nit: local vars normally have lower case first character. Maybe:
```suggestion
                    GenTree* broadcastScalar = hwintrinsic->Op(1);
```